src/ID3Helpers.js

```diff
@@ -128,20 +128,60 @@ function getFramesFromID3Body(ID3FrameBody, ID3Version, options = {}) {
             continue
         }
         const frameHeaderFlags = ID3Util.parseFrameHeaderFlags(bodyFrameHeader, ID3Version)
+        if(frameHeaderFlags.dataLengthIndicator) {
```

Comment:
I see `dataLenghtInicator` is only set for version 4, but apparently the version 3 supports it too according to the doc:
https://id3.org/id3v2.3.0#Frame_header_flags

https://github.com/Zazama/node-id3/blob/a6c784e8e0ecd3d4c32bae36042985199a229573/src/ID3Util.js#L191-L211

Reply:
Good catch, I will add it and add a v3 test